usaspending_api/api_contracts/contracts/award/idv/Funding.md

```diff
@@ -115,6 +115,19 @@ Returns File C financial data for an IDV (Indefinite Delivery Vehicle) award's d
                 }
             }
 
+# IDV Funding Roll-up [/api/v2/awards/idvs/funding-rollup/]
```

Comment:
The endpoints use underscores to separate words, so this would be `funding_rollup`